rpcs3/Emu/system_config_types.cpp

```diff
@@ -355,6 +355,21 @@ void fmt_class_string<move_handler>::format(std::string& out, u64 arg)
 	});
 }
 
+template <>
+void fmt_class_string<buzz_handler>::format(std::string& out, u64 arg)
+{
+	format_enum(out, arg, [](auto value) {
+		switch (value)
+		{
+		case buzz_handler::null: return "Null (use real Buzzers)";
+		case buzz_handler::one_controller: return "1 controller (1-4 players)";
+		case buzz_handler::two_controllers: return "2 controllers (5-7 players)";
```

Comment:
This will be the actual text written in the final config file, there's no need to be descriptive about it. In fact it just makes it harder for people looking at the log find errors with illegal entry values.

Reply:
What should i replace it with?

Reply:
Dropping the text sections in the parenthesses should suffice.